### DISCUSSION TITLE
1.2.0 beta 14 don’t discover instruments

### DISCUSSION THREAD

**Discussion by ciapamusche** [apohl79/audiogridder#1045] [Help]
Thank you for your application . I think is a game changer for music production community..

I discovered this plugin in these days so , I can commit a fault configuring it.

Downloading 1.2.0 on windows slave machine and 1.2.0 on Mac OSX Monterrey (using Logic Pro X and studio one 5) I had same problem:

Server load correctly all Plugins , DAWs recognise the server on plugin (AU,VST3) but if I press "+" No plugins are shown.

If I run last stable Version, it works only in studio one (VST3 and AU) showing all VSTs and I can load them with 24ms latency.

In Logic plugin istanze  doesn't load (Logic error message impossible to open plugin).
So is impossible to use AG in Logic. 
New beta don't load plugins and stable version don't load at all.

My network is a Wired LAN network with a Router and 2 LAN switch.

The signal has the following path:

SLAVE---Switch----Switch2----Router---Switch2---MASTER.

Any advice to solve the problem ?
Where I m wrong ?

Thank you

**Comment by apohl79** [comment]
Before investigating further, try to disable the firewall on windows  to see if it is a network issue.

**Reply by ciapamusche** [reply]
No luck.

Tried to install server 1.2.0 beta 14 (windows 10) but is stuck at loading plugin cache

**Reply by ciapamusche** [reply]
UPDATE: Deleting manually conf files and reloading the server installer on slave machine, the plugins are currently showed but is impossible to use them.

If I load for example a instrument (UJAM) on AG, my master MAC cpu Goes to 100%.
Tried to different buffer size no luck.

This happen with all plugins.

Advices?

**Reply by apohl79** [reply]
maybe create a short screen recording that shows your issue.
